### PR TITLE
Recognize .GRID/.FGRID File Types

### PR DESCRIPTION
**Pull request by bska**
The I/O code and simulator does not support this grid type, but we should nevertheless be able to convert between the formatted and the unformatted version of this file.

### CODE REVIEW

**Comment by bska** [comment]
jenkins build this please

**Comment by bska** [comment]
jenkins build this please

**Comment by bska** [comment]
jenkins build this please

File: test_util/convertECL.cpp [hunk 1]
```diff
@@ -356,30 +367,34 @@ int main(int argc, char **argv) {
...
-
+    }
+    else {
         auto search = to_binary.find(extension);
 
         if (search != to_binary.end()){
             resFile = rootN + search->second;
-        } else if (extension.substr(1,1)=="F"){
+        }
+        else if (extension.substr(1,1)=="F"){
```
Comment (akva2) [inline test_util/convertECL.cpp:391]: spacing while at it (and below)

Comment (bska) [reply test_util/convertECL.cpp]: > spacing while at it (and below)

Right.  I've pushed an update to that effect, including in a few other locations in the file.

**Comment by bska** [comment]
jenkins build this please

**Review by akva2 — APPROVED** [review]

**Pull request merged**